### PR TITLE
codespell: Add spelling check and fix all issues

### PR DESCRIPTION
**Pull request by mjtrangoni**
Hi,

this is a trivial PR, adding the [codespell](https://github.com/codespell-project/actions-codespell) tool to the CI, and fixing some non-code misspelling cases.

The CI file is called `linters.yml`, so that this can be extended in a future with other linter tools.

Best regards,

Mario

### CODE REVIEW

**Comment by ekoby** [comment]
Thank you for your contribution!
You can add yourself to CONTRIBUTORS file in this PR if you wish

File: tests/integ/bootstrap.exp [hunk 1]
```diff
@@ -18,7 +18,7 @@ spawn $ziti edge quickstart --home $env(QUICKSTART_HOME)
 set qs_id @spawn_id
 # wait until ER is connected
 expect {
-    "finished sychronizing api sessions" { }
```
Comment (ekoby) [inline tests/integ/bootstrap.exp:21]: this typo is needed as it is the output from another project

File: tests/integ/integ-test.exp [hunk 2]
```diff
@@ -12,12 +12,12 @@ spawn $ziti edge quickstart --already-initialized --home $env(QUICKSTART_HOME)
 set qs_id $spawn_id
 
 expect {
-    "finished sychronizing api sessions" { }
```
Comment (ekoby) [inline tests/integ/integ-test.exp:15]: same as above

**Review by ekoby — CHANGES_REQUESTED** [review]
see comments about expected output

File: tests/integ/bootstrap.exp (hunk 1 above)
Comment (ekoby) [reply tests/integ/bootstrap.exp:21]: if you also wish to fix that -- it is here:
https://github.com/openziti/ziti/blob/42f6e3fc487b70970b4b6f57b0af72ebf06a65df/router/handler_edge_ctrl/apiSessionAdded.go#L124

Comment (mjtrangoni) [reply tests/integ/bootstrap.exp:21]: @ekoby  thank you. I´ve created a PR there, see https://github.com/openziti/ziti/pull/1652

**Review by ekoby — APPROVED** [review]

**Pull request merged**